publish/tests/girder.py

```diff
@@ -13,7 +13,23 @@ class _GirderClientFolderFactory(factory.DictFactory):
 
 class _GirderClientDraftFolderFactory(_GirderClientFolderFactory):
     name = factory.Faker('numerify', text='#' * 6)
-    meta = factory.Dict({})
+    meta = factory.Dict(
+        {
+            'dandiset': factory.Dict(
+                {'name': factory.Faker('sentence'), 'description': factory.Faker('paragraph')}
+            )
+        }
+    )
+
+
+class _GirderClientInvalidDraftFolderFactory(_GirderClientDraftFolderFactory):
```

Comment:
I think this is better handled by [a Factory Boy `Trait`](https://factoryboy.readthedocs.io/en/latest/reference.html#traits). Since [referencing a sub-sub-factory](https://factoryboy.readthedocs.io/en/latest/reference.html#calling) within a `Trait` is not immediately intuitive (and because I wasn't sure it was actually going to work :smiley:), I tried implementing this myself, and pushed it as an additional commit.

Let me know what you think.

Reply:
Nevermind, `Trait` would be perfect, but it doesn't work properly in the disabled / `False` case. It's apparently [a known bug](https://github.com/FactoryBoy/factory_boy/issues/542).

Reply:
That's unfortunate, looks like it's been a bug for a while now too :frowning_face: 

At any rate, the test that relies on that behavior was removed in https://github.com/dandi/dandi-publish/pull/63/commits/7117a04d5ec021fc59e39c25f3d6539e39fce5b6 so it's not a problem anymore.